packages/cashier/src/components/empty-state/empty-state.tsx

```diff
@@ -1,22 +1,60 @@
-import { Icon, Text } from '@deriv/components';
 import React from 'react';
+import { Icon, Text, Button } from '@deriv/components';
 import './empty-state.scss';
 
-export type TEmptyStateProps = {
-    icon: string;
-    title: string | React.ReactNode;
-    description: string | React.ReactNode;
+type TAction = {
+    label: string;
+    onClick?: () => void;
+    primary?: boolean;
+    tertiary?: boolean;
+    disabled?: boolean;
 };
 
-const EmptyState = ({ icon, title, description }: TEmptyStateProps) => (
+export type TEmptyStateProps = {
+    icon?: string;
+    title?: string | React.ReactNode;
+    description?: string | React.ReactNode;
+    action?: TAction;
+} & ({ title: string | React.ReactNode } | { description: string | React.ReactNode });
```

Comment:
Suggestion: We have `TReactChildren` in cashier instead of `React.ReactNode`. It can be used instead.

Reply:
@hamid-deriv Will update 🙇🏻 
But what’s the reason for it? `React.ReactNode` is already a type alias, Why would we want to change the name and confuse developers? 🤔

Reply:
We have defined the `TReactChildren` in the package for having the same type name for children across the package and I just suggest that we have this type and you can use it.
However, I agree with you. I think we need to have a rule for using this. I think it's better to use `React.ReactNode` instead of `TReactChildren` and remove the `TReactChildren` from the cashier. As you said, it just confuses the developer.